Courses/Views/Screens/CourseView.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  CourseView.swift
+//  Courses
+//
+//  Created by Dmytro Churakov on 20.10.2022.
+//
+
+import SwiftUI
+
+struct CourseView: View {
+    var namespace: Namespace.ID
+    var course: Course = courses[0]
+    @Binding var show: Bool
+
+    var body: some View {
+        ZStack {
+            ScrollView {
+                cover
+                content
+                    .offset(y: 100)
+                    .padding(.bottom, 200)
+            }
+            .background(Color("Background"))
+            .ignoresSafeArea()
+            button
```

Comment:
I feel button name should be more descriptive. I have the same in my code following the course instructor, but seeing it here would have made more sense it it said something like dismissButton.

Reply:
@sghourab , agree, fixed